storage/google/cloud/storage/client.py

```diff
@@ -151,6 +151,24 @@ def current_batch(self):
         """
         return self._batch_stack.top
 
+    def get_service_account_email(self, project=None):
+        """Get the email address of the project's GCS service account
+
+        :type project: str
+        :param project:
+            (Optional) Project ID to use for retreiving service account email.
+            Defaults to the client's project.
```

Comment:
Project ID to use when retrieving the GCS service account email address.

Reply:
80805ec